pycryptobot.py

```diff
@@ -95,8 +95,12 @@ def calculateMargin(buy_size: float=0.0, buy_filled: int=0.0, buy_price: int=0.0
         print (f'sell_price: {sell_price}')
         print (f'sell_fee: {sell_fee}', "\n")
 
-    buy_value = buy_size + buy_fee
-    sell_value = sell_size - sell_fee
+    #buy_value = buy_size + buy_fee
+    #sell_value = sell_size - sell_fee
+    #profit = sell_value - buy_value
+
+    buy_value = buy_price + buy_fee
```

Comment:
Hi, i helped out Michael recently with the margin calculations.

Just wanted to ask why you think the the buy_value should be calculated using buy_price? 

buy_price is the actual price of the coin when bought, if it was BTC it could be £35k for example, it's not the size of the buy that was made.

Some changes were made yesterday that helped get the margins accurate and i have some further changes to add to a pull request so it will get better over time

Thanks
Mario

Reply:
I made some tests this morning on the pair i'm trading and margin was
totally wrong SOL/USDT was -91% while in real was -1.2%

Tested with over troubleshoot_margin.py as well, the first problem, in live
mode, was coming from the fee calculation, but also
the troubleshoot_margin.py was giving me a wrong value

BINANCE:

buy_size: 2.312
buy_filled: 104.913936
buy_price: 45.378
buy_fee: 0.104

sell_size: 2.2850475560844457
sell_price: 44.849
sell_fee: 0.09

buy_value: 2.416
sell_value: 2.195047556084446
profit: -0.22095244391555413

margin: -9.145382612398764


while in real it was:

BINANCE:

buy_size: 2.312
buy_filled: 104.913936
buy_price: 45.378
buy_fee: 0.104

sell_size: 2.2850475560844457
sell_price: 44.849
sell_fee: 0.09

buy_value: 45.482
sell_value: 44.75899999999999
profit: -0.7230000000000061

margin: -1.5896398575260675

in fact -1.6% was the real margin i had with manual calculations